core/src/main/java/com/datastax/oss/driver/internal/core/cql/CqlRequestHandler.java

```diff
@@ -660,6 +660,7 @@ private void processErrorResponse(Error errorMessage) {
         ThrottledAdminRequestHandler<ByteBuffer> reprepareHandler =
             ThrottledAdminRequestHandler.prepare(
                 channel,
+                true,
                 reprepareMessage,
                 repreparePayload.customPayload,
                 timeout,
```

Comment:
(this is the prepare-and-retry scenario when we get UNPREPARED)

Note that we reuse the same channel as the original query. If it is full, `reprepareHandler` will get an error when it calls `preAcquireId`.
In theory it would be better to reacquire from the pool in case there is another channel that is less busy. But that complicates the code here, and I don't think this is hugely important since the request will be retried on the next node anyway.